ApiTest.Business/Models/User.cs

```diff
@@ -0,0 +1,29 @@
+namespace ApiTest.Business.Models
+{
+    public class User
```

Comment:
Separate into different files pls, what if the number of models will raise - its not the best practice to store it in one file, espesially if they are independent models and add word Model in the class name